src/VisualStudio/Core/Def/Implementation/Interactive/CSharpResetInteractiveMenuCommand.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Editor;
+using Microsoft.VisualStudio.ComponentModelHost;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using System.ComponentModel.Design;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.Interactive
+{
+    internal sealed class CSharpResetInteractiveMenuCommand
```

Comment:
Why is this defined in the base assembly versus the C#/VB specific scripting assemblies?